frontend/src/components/PricingTable/PricingTable.vue

```diff
@@ -42,13 +42,10 @@
               </p>
 
               <div class="text-gray-400 h-5">
-                <span v-if="plan.type != PlanType.ENTERPRISE">
-                  {{ $t("subscription.per-instance") }}
+                <span v-if="plan.type != PlanType.FREE">
```

Comment:
TEAM plan has per-month billing.

We want to lower the bar for user to TEAM Plan.

Enterprise plan is annual billing.